### ISSUE TITLE
shp.Create always removes last three characters of filename

### ISSUE THREAD

**Issue by gertcuykens** [jonas-p/go-shp#26]
```go
layer, err := shp.Create("layer/test.shp", shp.POLYLINE)
```
works but doing 

```go
layer, err := shp.Create("layer/test", shp.POLYLINE)` instead results in something unexpected:
```

```
teshp
teshx
tedbf
```

**Comment by fawick** [comment]
Thanks for your report, I think I know what's going on here. I hope to have a fix for this tonight.

**Comment by fawick** [comment]
Fixed with 215a5781289c857f70b19c2740e65d81de343b50.

**Comment by fawick** [comment]
Released 215a5781289c857f70b19c2740e65d81de343b50 as v0.1.1
